packages/manager/apps/hpc-vmware-managed-vcd/public/translations/terminate/Messages_fr_FR.json

```diff
@@ -0,0 +1,4 @@
+{
+  "terminate_managed_vcd_success": "Votre demande de résiliation a bien été prise en compte. Un e-mail contenant la procédure vous a été envoyé.",
+  "terminate_managed_vcd_ftc_success": "Votre demande de résiliation a bien été prise en compte. Un e-mail de confirmation vous a été envoyé."
```

Comment:
```suggestion
  "terminate_managed_vcd_ftc_success": "Votre demande de résiliation a bien été prise en compte. Un email de confirmation vous a été envoyé."
```

Reply:
Can be put on common-translation ? May be create a ticket for refactor that later ? Or add in actual ticket to centralize common content